build.gradle

```diff
@@ -37,6 +37,7 @@ subprojects {
   license {
     header = file("${rootDir}/gradle/license/LICENSE_HEADER")
     ext.year = Calendar.getInstance().get(Calendar.YEAR)
```

Comment:
We probably need to hard-code this to 2018.

Reply:
@wmoustafa @shardulm94 Could you confirm if we need to hard-code it or not? I can change it accordingly

Reply:
I will get back on this.

Reply:
Okay I checked. For any new files or updated files in 2019, the year should be 2019. So, please keep it to Calendar.getInstance().get(Calendar.YEAR), and update the year in this PR manually to 2019.